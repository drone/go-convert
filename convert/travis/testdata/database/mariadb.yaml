# https://docs.travis-ci.com/user/database-setup/#mariadb

addons:
  mariadb: '10.0'