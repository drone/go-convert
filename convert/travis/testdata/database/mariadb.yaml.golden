kind: pipeline
spec:
  stages:
  - desc: converted from travis.yml
    name: pipeline
    spec:
      steps:
      - name: mariadb
        spec:
          envs:
            MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: "yes"
            MARIADB_DATABASE: test
            MARIADB_USER: root
            MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
            MYSQL_DATABASE: test
            MYSQL_USER: root
          image: mariadb
          ports:
          - "3306"
        type: background
      - name: install
        spec:
          run: bundle install --jobs=3 --retry=3
        type: script
      - name: script
        spec:
          run: rake
        type: script
    type: ci
version: 1
